Modules/Core/Common/include/itkResourceProbe.hxx

```diff
@@ -427,8 +427,6 @@ template <typename ValueType, typename MeanType>
 void
 ResourceProbe<ValueType, MeanType>::JSONReport(std::ostream & os)
 {
-  const std::stringstream ss;
```

Comment:
@hjmjohnson  Another case that I found because you added those const keywords (PR https://github.com/InsightSoftwareConsortium/ITK/pull/5035 commit https://github.com/InsightSoftwareConsortium/ITK/commit/375ee8c47a03792d0d3d5447c6b4c4be6b9f2fb4)! A local const stringstream looks suspicious, doesn't it?!? When it was still non-const, this unused variable could remain under the radar, for more than seven years, actually!

Reply:
Great!  Linting tools often have un-intended benefits.